pkg/virtual/workspaces/registry/org.go

```diff
@@ -23,50 +23,77 @@ import (
 	tenancyclient "github.com/kcp-dev/kcp/pkg/client/clientset/versioned/typed/tenancy/v1alpha1"
 	workspaceinformer "github.com/kcp-dev/kcp/pkg/client/informers/externalversions/tenancy/v1alpha1"
 	frameworkrbac "github.com/kcp-dev/kcp/pkg/virtual/framework/rbac"
-	workspaceauth "github.com/kcp-dev/kcp/pkg/virtual/workspaces/auth"
+	workspaceauth "github.com/kcp-dev/kcp/pkg/virtual/workspaces/authorization"
 )
 
-// CreateAndStartOrg creates an Org struct that contains all the required clients and caches to retrieve user workspaces inside an org
+// CreateAndStartOrg creates an Org that contains all the required clients and caches to retrieve user workspaces inside an org
 // As part of an Org, a WorkspaceAuthCache is created and ensured to be started.
-func CreateAndStartOrg(orgRBACClient rbacv1client.RbacV1Interface, orgClusteWorkspaceClient tenancyclient.ClusterWorkspaceInterface, orgRBACInformers rbacinformers.Interface, orgCRBInformer rbacinformers.ClusterRoleBindingInformer, orgClusterWorkspaceInformer workspaceinformer.ClusterWorkspaceInformer) *Org {
+func CreateAndStartOrg(
+	orgRBACClient rbacv1client.RbacV1Interface,
+	orgClusteWorkspaceClient tenancyclient.ClusterWorkspaceInterface,
+	orgRBACInformers rbacinformers.Interface,
+	orgCRBInformer rbacinformers.ClusterRoleBindingInformer,
+	orgClusterWorkspaceInformer workspaceinformer.ClusterWorkspaceInformer,
+) *Org {
 	orgSubjectLocator := frameworkrbac.NewSubjectLocator(orgRBACInformers)
-	orgReviewerProvider := workspaceauth.NewAuthorizerReviewerProvider(orgSubjectLocator)
+	orgReviewer := workspaceauth.NewReviewer(orgSubjectLocator)
 
 	orgWorkspaceAuthorizationCache := workspaceauth.NewAuthorizationCache(
 		orgClusterWorkspaceInformer.Lister(),
 		orgClusterWorkspaceInformer.Informer(),
-		orgReviewerProvider.ForVerb("get"),
+		orgReviewer,
+		*workspaceauth.NewAttributesBuilder().
+			Verb("get").
+			Resource(tenancyv1alpha1.SchemeGroupVersion.WithResource("clusterworkspaces"), "workspace").
+			AttributesRecord,
 		orgRBACInformers,
 	)
 
 	newOrg := &Org{
-		rbacClient:                orgRBACClient,
-		crbInformer:               orgCRBInformer,
-		crbLister:                 orgCRBInformer.Lister(),
-		workspaceReviewerProvider: orgReviewerProvider,
-		clusterWorkspaceClient:    orgClusteWorkspaceClient,
-		clusterWorkspaceLister:    orgWorkspaceAuthorizationCache,
-		stopCh:                    make(chan struct{}),
-		authCache:                 orgWorkspaceAuthorizationCache,
+		rbacClient:             orgRBACClient,
+		crbInformer:            orgCRBInformer,
+		crbLister:              orgCRBInformer.Lister(),
+		workspaceReviewer:      orgReviewer,
+		clusterWorkspaceClient: orgClusteWorkspaceClient,
+		clusterWorkspaceLister: orgWorkspaceAuthorizationCache,
+		stopCh:                 make(chan struct{}),
+		authCache:              orgWorkspaceAuthorizationCache,
 	}
 
 	newOrg.authCache.Run(1*time.Second, newOrg.stopCh)
 
 	return newOrg
 }
 
+func RootOrg(
```

Comment:
nit: NewRootOrg

Reply:
fixed in commit 56947bcb00c6b1746f053060eb32042705a133a2